pkg/podrescli/client.go

```diff
@@ -33,6 +35,48 @@ func (ci *ContainerIdent) String() string {
 	return fmt.Sprintf("%s/%s/%s", ci.Namespace, ci.PodName, ci.ContainerName)
 }
 
+func (ci *ContainerIdent) Set(val string) error {
```

Comment:
not a fan of these interface leaking into utility packages. Do we have other options?

Reply:
Yes we do:
1. Use additional helper string variable and drop this ```Set``` function
2. move the logic into somewhere else?
BTW what is the reason that you don't like interface leaking into utility packages?

Reply:
> BTW what is the reason that you don't like interface leaking into utility packages?
Separation of concerns, minimization of inter-package deps/more loose coupling.
In this specific case I don't see (yet) many gains about using an intermediate variable, but we add a dep between packages. Doesn't seem an obvious gain to me.

Reply:
> In this specific case I don't see (yet) many gains about using an intermediate variable

In the ```main.go``` we can do something like:
```
flags.StringVar(&contRef, "container-reference", "", "usage description") 
ci := ContainerIdentFromString(contRef)
```
and we don't need the ```Set``` function anymore, so you eliminate the interface leaking into utility package

Reply:
that seems fine.